### ISSUE TITLE
Entering image descriptions is needlessly awkward

### ISSUE THREAD

**Issue by chucker** [chucker/Mastonaut#18]
You're in the new toot window (1), then add an image, which has a little thumbnail, where you click on a bubble (2), and that opens a popover (3), which also has a way too narrow text field.

Surely we can make that nicer.

I think Tusker does this better. Rather than arranging thumbnails in a horizontal stack, it places them vertical, leaving room to the right of them for a description each.

**Comment by chucker** [comment]
I think I want this for 1.5.

**Comment by uliwitness** [comment]
While you're at it: Would be cool if it presented a warning sheet when you try to post and any image attachment is missing alt text. Just a simple "3 out of 4 images are missing their image descriptions, do you really want to post like this?" [Edit] [Post without] sheet would be fine. Bonus points if it selects the first empty description for editing.

**Comment by chucker** [comment]
I think I'm punting on this yet again.

**Comment by stuffmc** [comment]
1.5, you said? :-) ;-) ;-) Yeah sorry I really should contribute to the project :( I hope to have more time in front of the computer in a few weeks when so much of my "actual" work (in the house) is done.

**Comment by chucker** [comment]
Yeah, it’s among the features I definitely want to do, but haven’t really gotten around to.
